c_src/ffmpeg_video_compositor/filter.c

```diff
@@ -1,17 +1,17 @@
 #include "filter.h"
 /**
  * @brief Append a header of the filter description to the string (buffer).
- * Creates two input pads with output pads named [in_1] and [in_2]
+ * Creates \p n_videos input nodes with output pads named [in_1], [in_2], ..,
+ * [in_ \p n_videos].
  *
  * @param filters_str Description destination string (buffer)
  * @param filters_size Remaining size of the buffer
- * @param width Width of the videos
- * @param height Height of the videos
- * @param pixel_format Pixel format code of the videos
- * @return Number of characters written to the buffer
+ * @param videos Array of input videos.
+ * @param n_videos Size of the videos array
+ * @return int
  */
-static int init_filters_string(char *filters_str, int filters_size, int width,
-                               int height, int pixel_format);
+static int init_filters_string(char *filters_str, int filters_size,
```

Comment:
Maybe a `get_filters_string` would be a better name? Or something like `append_input_nodes`?

Reply:
Changed to `append_input_nodes_filters_string`.